packages/evm/src/evm.ts

```diff
@@ -574,7 +721,9 @@ export class EVM implements EVMInterface {
       codeAddress: message.codeAddress,
       gasRefund: message.gasRefund,
       containerCode: message.containerCode,
+      chargeCodeAccesses: message.chargeCodeAccesses,
       blobVersionedHashes: message.blobVersionedHashes ?? [],
+      accessWitness: message.accessWitness,
```

Comment:
Can we make these (so: `chargeCodeAccesses` and `accessWitness` for `env`, and `accessWitness` for `Message` below) optional for now, and then do `!` within `if (Verkle EIP activated) { ... }` blocks ?

I know this is rather internal, but people are muddling a lot with and go deep into our EVM code, and this would have a higher chance to not break anything for them.

Reply:
Update: ah, seeing that for Message it (at least partly? not sure about `chargeCodeAccesses`) seems to be. Still for `env` would be also nice if easily doable.

Reply:
i am confused, both are optional in env and message